src/modules/commander/commander_params.c

```diff
@@ -802,6 +802,21 @@ PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
  * Temporary Parameter to enable interface testing
  *
  * @boolean
+ * @reboot_required true
  * @group Mission
  */
-PARAM_DEFINE_INT32(COM_OBS_AVOID, 0);
\ No newline at end of file
+PARAM_DEFINE_INT32(COM_OBS_AVOID, 0);
+
+/**
+ * Set onboard controller bootup timeout
+ *
+ * This parameter defines the bootup timeout.
+ * After the timeout a mavlink message to warn the user that the system
+ * is still booting up is triggered.
```

Comment:
The purpose of this param is not really clear to me. From what I read, the only change is the time of when the mavlink statustext warning is sent. However, it's not clear if this warning is now always triggered even if no onboard computer is used at all.

Also, I would probably call it an onboard computer or companion computer, and not an `onboard controller` because it's not really controlling anything, right?

And it probably requires a reboot. :smile:

Reply:
I think I am losing something here:
why should "the warning always be triggered even if no onboard computer is not used at all"?

That message is only printed if OA is enabled (at the current time onboard controller is only used for OA), AFAIK.